### PR TITLE
Issue 34: Bring over columns block

### LINKED ISSUE #34: Columns
- Move over columns from https://github.com/hlxsites/caesars/tree/main/caesars-palace/blocks/columns

### PR DESCRIPTION
**Pull request by jfait**
Fix #34

## 🔗 Test URLs:

- Before: https://main--caesars-careers--hlxsites.hlx.page/block-library/blocks/authoringguide-columns
- After: https://issue-34-bringOverColumnsBlock--caesars-careers--hlxsites.hlx.page/block-library/blocks/authoringguide-columns

## 📝 Description:

Bring over columns block. 
Reuse block library as test content.
May need to prune columns later to pull stuff out that's not needed (assuming it will never be useful)

### CODE REVIEW

File: careers/blocks/columns/columns.css [hunk 1]
```diff
@@ -1,33 +1,336 @@
+.section.columns-container .default-content-wrapper {
+    text-align: center;
+}
+
 .columns > div {
-  display: flex;
-  flex-direction: column;
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 64px;
```
Comment (tmathern) [inline careers/blocks/columns/columns.css:8]: Use a var?

File: careers/blocks/columns/columns.css [hunk 2]
```diff
@@ -1,33 +1,336 @@
...
+    text-align: justify;
 }
 
 .columns img {
+    width: 100%;
+}
+
+.columns.product-quickfacts.stretch-end {
   width: 100%;
+  margin-bottom: 32px;
```
Comment (tmathern) [inline careers/blocks/columns/columns.css:25]: Use vars?

File: careers/blocks/columns/columns.css [hunk 3]
```diff
@@ -1,33 +1,336 @@
...
+  font-size: 14px;
```
Comment (tmathern) [inline careers/blocks/columns/columns.css:26]: Use vars?

File: careers/blocks/columns/columns.css [hunk 4]
```diff
@@ -1,33 +1,336 @@
...
+.columns.columns-2-cols.cet-card > div > div {
+    text-align: unset;
+}
+
+.columns.product-quickfacts.stretch-end > div > div {
+  flex: 1;
+  display: flex;
+  justify-content: center;
+  font-family: var(--body-font-family);
+  font-size: 16px;
```
Comment (tmathern) [inline careers/blocks/columns/columns.css:54]: Use vars?

File: careers/blocks/columns/columns.css [hunk 5]
```diff
@@ -1,33 +1,336 @@
...
+}
+
+.columns.product-quickfacts.stretch-end img {
+  max-height: 100px;
+  max-width: 200px;
+  margin-bottom: 32px;
```
Comment (tmathern) [inline careers/blocks/columns/columns.css:60]: Use vars?

File: careers/blocks/columns/columns.css [hunk 6]
```diff
@@ -1,33 +1,336 @@
...
+  margin-top: 24px;
```
Comment (tmathern) [inline careers/blocks/columns/columns.css:61]: Use vars?

File: careers/blocks/columns/columns.css [hunk 7]
```diff
@@ -1,33 +1,336 @@
...
+    }
+
+    .columns [data-align='right'] {
+        text-align: right;
+    }
+
+    .columns > div {
+        display: flex;
+        flex-flow: row nowrap;
+        gap: 32px;
```
Comment (tmathern) [inline careers/blocks/columns/columns.css:138]: Use vars?

File: careers/blocks/columns/columns.css [hunk 8]
```diff
@@ -1,33 +1,336 @@
...
+        display: none;
+    }
+
+    .columns.columns-2-cols.intro > div:first-child {
+        justify-content: unset;
+    }
+
+    .columns.product-quickfacts.stretch-end {
+      width: 80%;
+      margin-bottom: 32px;
```
Comment (tmathern) [inline careers/blocks/columns/columns.css:200]: Use vars?

**Review by tmathern — CHANGES_REQUESTED** [review]

**Review by tmathern — CHANGES_REQUESTED** [review]
Try to use more vars. All those sizes seem related to a site size template.

File: careers/blocks/columns/columns.css (hunk 4 above)
Comment (jfait) [reply careers/blocks/columns/columns.css:54]: no body font for 16

Comment (jfait) [reply careers/blocks/columns/columns.css:54]: no body font for 16

**Review by tmathern — APPROVED** [review]

**Pull request merged**